test/problems/sampling/protein_folding/resources/test_contact_map_builder/test_create_new_qubit_list_1

```diff
@@ -0,0 +1,2 @@
+0.5 * IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
+- 0.5 * IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIZIIII
```

Comment:
I assume this is something you test against. Wouldn't it make more sense to construct the expected op from a sparse label? This file does not really add any benefit and having the expected op constructed from a sparse label would very much improve readability.

The same applies to similar files below

Reply:
I agree. I have recently fixed a bug that resulted in having way more qubit registers and now it is very tricky to eyeball it and understand the meaning of each index. Still kept the files because it is a lot of manual work to switch to sparse but it is definitely something that should be done eventually.